client/src/components/previews/ReportPreview.js

```diff
@@ -0,0 +1,335 @@
+import API from "api"
+import { gql } from "apollo-boost"
+import { parseHtmlWithLinkTo } from "components/editor/LinkAnet"
+import * as FieldHelper from "components/FieldHelper"
+import Fieldset from "components/Fieldset"
+import LinkToNotPreviewed from "components/LinkToNotPreviewed"
+import PlanningConflictForReport from "components/PlanningConflictForReport"
+import Tag from "components/Tag"
+import { Field, Form, Formik } from "formik"
+import { Person, Report, Task } from "models"
+import moment from "moment"
+import ReportPeople from "pages/reports/ReportPeople"
+import PropTypes from "prop-types"
+import React from "react"
+import Settings from "settings"
+import utils from "utils"
+
+const GQL_GET_REPORT = gql`
+  query($uuid: String!) {
+    report(uuid: $uuid) {
+      uuid
+      intent
+      engagementDate
+      duration
+      atmosphere
+      atmosphereDetails
+      keyOutcomes
+      nextSteps
+      cancelledReason
+      reportText
+      releasedAt
+      state
+      location {
+        uuid
+        name
+        lat
+        lng
+      }
+      authors {
+        uuid
+        name
+        rank
+        role
+        avatar(size: 32)
+      }
+      reportPeople {
+        uuid
+        name
+        author
+        primary
+        attendee
+        rank
+        role
+        status
+        avatar(size: 32)
+        position {
+          uuid
+          name
+          type
+        }
+      }
+      principalOrg {
+        uuid
+        shortName
+        longName
+        identificationCode
+        type
+      }
+      advisorOrg {
+        uuid
+        shortName
+        longName
+        identificationCode
+        type
+      }
+    }
+  }
+`
+
+const ReportPreview = ({ className, uuid, previewId }) => {
+  const { data, error } = API.useApiQuery(GQL_GET_REPORT, {
+    uuid
+  })
+
+  if (!data) {
+    if (error) {
+      return <p>Could not load the preview</p>
+    }
+    return null
+  }
+
+  let report
+
+  data.report.cancelled = !!data.report.cancelledReason
+  data.report.reportTags = (data.report.tags || []).map(tag => ({
```

Comment:
`report.tags` doesn't exist in gql.

Reply:
Hmm yes, it might be better if the tasks are shown in the preview